docs/release/trg-4/trg-4-06.md

```diff
@@ -148,9 +131,9 @@ You have to adapt some of the provided links to match your used version.
 - GitHub project: [https://github.com/dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker)
 - DockerHub: [https://hub.docker.com/_/microsoft-dotnet-aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet)
 
-### Linux
+### Hashicorp Vault image
```

Comment:
I added this, but at least I don't need it anymore as base images are just relevant for distribution purposes and NOT for local testing purposes.

AFAIK the EDC team once distributed images with a vault. But likely not as a base image. @paullatzelsperger or @wolf4ood can you confirm?

Reply:
In this case its just a mentioning of possibility. But i can remove it.

Reply:
I think the change regarding the vault image should be revert (and for sure not replace the linux alpine image) as  I assume the image @tom-rm-meyer-ISST is referring to also uses temurin as base: https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-dataplane/edc-dataplane-hashicorp-vault/src/main/docker/Dockerfile#L23

Reply:
@evegufy i'm not sure; what you reference has nothing to do with vault besides the name of it. But yes @tom-rm-meyer-ISST whats your status with the vault usage?

Reply:
@tom-rm-meyer-ISST vault is gone